index.js

```diff
@@ -0,0 +1,18 @@
+module.exports = (api, options) => {
```

Comment:
See https://cli.vuejs.org/dev-guide/plugin-dev.html#service-plugin for what this index.js file is